README.md

```diff
@@ -24,15 +23,25 @@ The `pulumilocal` command has the same usage as the `pulumi` command. For detail
 please refer to the man pages of `pulumi -h`.
 
 For example:
-### Create a new Pulumi Project with Stack name lsdev
+
+### Add environment variables to store state on local backend (optional)
 ```shell
-mkdir myproj
 export PULUMI_CONFIG_PASSPHRASE=lsdevtest
 export PULUMI_BACKEND_URL=file://`pwd`/myproj
+```
+_Note: For further options please consult the official documentation on available [environment variables][env_vars] and [local backend][local_backend]._
+
+[env_vars]: https://www.pulumi.com/docs/cli/environment-variables/
```

Comment:
Wow, wasn't aware of this markdown syntax to reference links! (usually always used the format `[link text](http://...)`). Nice one! :)

Reply:
Glad you like it, thought it provides better markdown readability than dropping it inline :)